### PR TITLE
Directorize and alphabetize nodes

### PR DESCRIPTION
**Pull request by ivov**
This PR places all nodes in their own dirs, and alphabetizes the listings of nodes and creds. Needed for i18n, which will allow for a `/translations` dir in each node dir.

Submitted now to prevent that changes in `master` to files that are relocated in the i18n branch cause large merge conflicts when merging `master` to keep the i18n branch up to date.

### CODE REVIEW

File: packages/nodes-base/package.json [hunk 1]
```diff
@@ -231,41 +229,42 @@
...
       "dist/credentials/RundeckApi.credentials.js",
+      "dist/credentials/S3.credentials.js",
       "dist/credentials/SalesforceJwtApi.credentials.js",
       "dist/credentials/SalesforceOAuth2Api.credentials.js",
       "dist/credentials/SalesmateApi.credentials.js",
+      "dist/credentials/SalesmateApi.credentials.js",
       "dist/credentials/SeaTableApi.credentials.js",
       "dist/credentials/SecurityScorecardApi.credentials.js",
       "dist/credentials/SegmentApi.credentials.js",
+      "dist/credentials/SegmentApi.credentials.js",
```
Comment (mutdmour) [inline packages/nodes-base/package.json:240]: does this need to be duplicated?

File: packages/nodes-base/package.json [hunk 2]
```diff
@@ -231,41 +229,42 @@
...
       "dist/credentials/SendyApi.credentials.js",
       "dist/credentials/SentryIoApi.credentials.js",
-      "dist/credentials/SentryIoServerApi.credentials.js",
       "dist/credentials/SentryIoOAuth2Api.credentials.js",
+      "dist/credentials/SentryIoServerApi.credentials.js",
       "dist/credentials/ServiceNowOAuth2Api.credentials.js",
+      "dist/credentials/Sftp.credentials.js",
       "dist/credentials/ShopifyApi.credentials.js",
       "dist/credentials/Signl4Api.credentials.js",
+      "dist/credentials/Signl4Api.credentials.js",
```
Comment (mutdmour) [inline packages/nodes-base/package.json:250]: does this need to be duplicated?

**Review by mutdmour — COMMENTED** [review]
cannot add comments to moved files..
1. Should not we just delete `ActivationTrigger.node.json`
2. nitpick - maybe rename `itemLists.svg` to `ItemLists.svg`
3. nitpick - maybe rename `n8nTrigger.svg` to `N8nTrigger.svg`
4. nitpick - maybe rename `RespondToWebhook/webhook.svg` and `Webhook/webhook.svg`
5. we should delete `RunAt` node if it's not being used

File: packages/nodes-base/package.json [hunk 3]
```diff
@@ -500,30 +500,30 @@
       "dist/nodes/Microsoft/ToDo/MicrosoftToDo.node.js",
       "dist/nodes/Mindee/Mindee.node.js",
       "dist/nodes/Misp/Misp.node.js",
-      "dist/nodes/MonicaCrm/MonicaCrm.node.js",
-      "dist/nodes/MoveBinaryData.node.js",
       "dist/nodes/Mocean/Mocean.node.js",
       "dist/nodes/MondayCom/MondayCom.node.js",
       "dist/nodes/MongoDb/MongoDb.node.js",
+      "dist/nodes/MonicaCrm/MonicaCrm.node.js",
+      "dist/nodes/MoveBinaryData.node.js",
```
Comment (mutdmour) [inline packages/nodes-base/package.json:507]: need to update this

**Comment by ivov** [comment]
Thanks, comments addressed. Jan confirmed `RunAt` can be removed.

Retested with this hard reset and fixed a couple more filepaths:

```sh
git reset --hard; git clean -dffx
npm i; lerna bootstrap --hoist; npm run build
```

About 2/3/4, the convention is camelCase for svg/png filenames.

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with `n8n@0.150.0`

**Pull request merged**